src/coreclr/debug/runtimeinfo/contractpointerdata.cpp

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include "common.h"
+
+#include <stddef.h>
+#include <stdint.h>
+
+#include "threads.h"
+
+extern "C"
+{
+  
+extern const uintptr_t contractDescriptorPointerData[];
```

Comment:
Is this declaration needed?

Reply:
interestingly, yes.  without it clang doesn't emit the constant into the object file at all